Automattic/wp-calypso
# Jetpack monthly plans european test

The a/b test shipped with https://github.com/Automattic/wp-calypso/pull/30766 had some bugs that this PR tries to solve. There was a race condition that caused most of the users were assigned an 'US' location by default, so they would skip the test.

With these changes, we don't show the plans grid until the geolocation info is already fetched, so we are sure the test doesn't start until all the info is ready.

Test instructions are still the same:

0. Add `"jetpackMontlyPlansOnly_20190312":"monthlyOnly"` in Chrome's dev tools > Application > Local Storage > Calypso URL > ABTests key
1. Go to http://calypso.localhost:3000/jetpack/connect/store. You shouldn't see the yearly/monthly toggle and should get the monthly plan grid by default

![image](https://user-images.githubusercontent.com/1554855/54216312-b5392280-44e9-11e9-8e6f-6fcf7d73d028.png)
